### DISCUSSION TITLE
两个27寸作为上下屏看视频，能否冻结窗格方式悬浮视频框，方便边看视频边下滑看评论

### DISCUSSION THREAD

**Discussion by yimingriyue** [the1812/Bilibili-Evolved#4489] [功能建议 / Ideas]
如题。个人喜欢大屏竖起来用，32寸作竖屏放直播，上下两个27寸显示器，正好浏览器铺满，方便一览无遗。

**Comment by Geeyun-JY3** [comment]
好像有点像 #4180

**Reply by yimingriyue** [reply]
是的，如bilibili网页，下滑页面时，页面顶部的吸顶分区，把视频播放部分，有吸顶功能。

**Comment by yimingriyue** [comment]
Tampermonkey，新增脚本 Bilibili-Player框架固定



// ==UserScript==
// @name         Bilibili-Player框架固定
// @namespace    your-namespace
// @version      1.0
// @description  Sticky player frame on Bilibili website
// @author       Your Name
// @match        https://www.bilibili.com/*
// @grant        GM_addStyle
// ==/UserScript==


// 将Bilibili-Player框架固定
window.onload = function() {
  var player = document.querySelector('#bilibili-player');
  player.style.position = 'fixed';
  player.style.top = '150px'; // 根据需要调整距离下移的高度
  player.style.left = '150px'; // 根据需要调整距离左侧的距离
  player.style.width = '80%'; // 缩小左右范围至视频宽度的80%
  player.style.zIndex = '9999'; // 设置相对高度，确保覆盖其他元素
};
